internal/internal_activity.go

```diff
@@ -181,13 +181,13 @@ func getValidatedActivityOptions(ctx Context) (*activityOptions, error) {
 		p.TaskListName = p.OriginalTaskListName
 	}
 	if p.ScheduleToStartTimeoutSeconds <= 0 {
-		return nil, errors.New("missing or negative ScheduleToStartTimeoutSeconds")
+		return nil, errors.New("invalid non-positive ScheduleToStartTimeoutSeconds")
```

Comment:
I think you shouldn't change this. It's only ScheduleToCloseTimeoutSeconds' message that is incorrect.